roles/srobo-nginx/templates/nginx.conf

```
@@ -230,6 +230,9 @@ http {
     rewrite ^/pbv4 /docs/kit/power_board redirect;
     rewrite ^/sbv4 /docs/kit/servo_board redirect;
 
+    # We now use Discord instead of a self hosted forum
+    rewrite ^/forums /docs/team_admin/discord redirect;
```

Issue: Looking at the commented out code (which should probably be removed), `/forum` was the path
```suggestion
    rewrite ^/forum /docs/team_admin/discord redirect;
```

---

I got `/forums` from the current interactive troubleshooter. Maybe we should do both?

---

Makes sense to me!

---

Note: [`/forum/`](https://studentrobotics.org/forum/) does already redirect, we should probably tidy that.

---

fc6b1dde8ad4b792d4b0625c4d0ad5e98d435fa2 solves all of these.